src/trust_is_risk.js

```diff
@@ -14,10 +14,46 @@ var TrustDB = require("./trust_db");
 var DirectTrust = require("./direct_trust");
 
 class TrustIsRisk {
-  node : bcoin$FullNode
+  node : (bcoin$FullNode | bcoin$SPVNode)
   db : TrustDB
 
-  constructor(node : bcoin$FullNode) {
+  fakePubKeyArray : Array<number>
+  fakePubKey : Buffer
+  fakeKeyRing : KeyRing
+  tag : Buffer
+
+  compressedFakePubKeyArray : Array<number>
+  compressedFakePubKey : Buffer
+  compressedFakeKeyRing : KeyRing
+  compressedTag : Buffer
+
+  constructor(node : (bcoin$FullNode | bcoin$SPVNode)) {
+    this.fakePubKeyArray = [0x04,            // constant 0x04 prefix
```

Comment:
Put the constants out of the class. Use corresponding `--harmony` flags

Reply:
@dionyziz There's a million flags, I went one time through them all but they seem to do something else entirely. I read a bit about ESnext and I understood that's a completely separate transpiler (?) that we're currently not using. Can you give me a pointer?

Reply:
Managed to do it without new flags. Check if the result is OK.